CMakeLists.txt

```diff
@@ -46,6 +48,10 @@ if(USE_SNMALLOC_STATS)
   add_definitions(-DUSE_SNMALLOC_STATS)
 endif()
 
+if(CACHE_FRIENDLY_OFFSET)
```

Comment:
Given the tests are setting this manually.  Should this just be added to the `add_shim` macro?

Reply:
Done